server/serviceClients/authClient/local.js

```diff
@@ -0,0 +1,83 @@
+// This is an example of the simplest scenario when the widget is hosted and used in the same secure domain
+// where as the provider credentials are safe to be stored in the widget server, hence as soon as the auth token is valid
+// Or no auth token needed at all, 
+// simply return the provider credentials, coppied from : 
+// const providerConfig = require('../../../../example/application/configuration')
+
+const {decrypt} = require('../../utils');
+const config = require('../../config');
+
+export class AuthClient {
+  token;
+  constructor(token){
+    this.token = token;
+  }
+
+  async getSecretExchange(iv){
+
+    const key = Buffer.from(config.LocalAuthEncryptionKey, 'base64').toString('hex')
+    const token = decrypt(this.token, key, iv);
+    // if token is valid, or use your own way to validate the token.
+    if(token === config.LocalAuthPhrase){ 
+      return {
+        mxInt: {
+          username: process.env.MxClientId,
+          password: process.env.MxApiSecret,
+          basePath: 'https://int-api.mx.com',
+          vcEndpoint: 'https://int-api.mx.com/',
+          provider: 'mx_int',
+          available: true
+        },
+        mxProd: {
+          username: process.env.MxClientIdProd,
```

Comment:
It looks like you added several variables to the .env file. These should be added to .env.example. It may make sense to group the things that are only needed with local auth together under a comment describing when you would actually need them